docker/cpp-root.dockerfile

```diff
@@ -1,18 +1,13 @@
-FROM faasm/faabric-base:0.3.4
-
-RUN apt-get update
-RUN apt-get install -y software-properties-common
-# Make sure packages from faabric-base are up-to-date
-RUN apt-get upgrade -y
-
-# Faasm-specific dependencies
-RUN apt-get install -y \
-    cgroup-tools \
-    iproute2 \
-    iptables \
-    libcgroup-dev
-
-RUN apt-get clean autoclean
-RUN apt-get autoremove
-
-CMD /bin/bash
+FROM faasm/faabric-base:0.3.5
+
+# Install Faasm-specific APT dependencies
+RUN apt update \
+    && apt upgrade -y \
+    && apt install -y \
+        cgroup-tools \
+        iproute2 \
+        iptables \
+        libcgroup-dev \
+        software-properties-common \
+    && apt clean autoclean \
```

Comment:
`apt clean autoclean` and `apt autoremove` both need the `--yes/-y` flag too, they can ask for user input sometimes